extensions/cornerstone-dynamic-volume/src/getHangingProtocolModule.ts

```diff
@@ -280,7 +277,8 @@ function getFusionViewports() {
 }
 
 const defaultProtocol = {
-  id: 'default4D',
+  id:
```

Comment:
i don't think you should make the full pathname here, if you say 'default' it automatically should create the  `@ohif/extension-cornerstone-dynamic-volume.hangingProtocolModule.default` if i'm not mistaken

Reply:
I changed the id when I was comparing it with TMTV HP and I saw that one using `@ohif/extension-tmtv.hangingProtocolModule.ptCT` as id

Reply:
does it work if you say `default` here? if so please do so

Reply:
It works if we set it to default because it would use the default HP from default mode but I've reverted my change renaming the HP back to `default4D`